### PR TITLE
Engage refactors : add user stories narrative : v1.0

### PR DESCRIPTION
**Pull request by engagetech**
Creates a user stories centered narratives.
Adds an extra user story for the "extra credits" section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,22 +2,91 @@ Goal
...
+> 1. _The date of the expense_
+> 2. _The VAT (Value added tax) associated to this expense. VAT is the UK’s sales tax. It is 20% of the value of the expense, and is included in the amount entered by the user._
+> 3. _The reason of the expense_
+>
+
+
+
+**User story 2:**
+
+> **As a user, i want to be able to enter expenses in the UI and have them saved in the database.**
```
Comment (engagetech) [inline README.md]: I think US 2 and US1 should be inverted.
The VAT thing, should only come in a second part of the work and so, would be better situated in the second user story: -jmt

File: README.md [hunk 2]
```diff
@@ -2,22 +2,91 @@ Goal
...
+By email, the front end developer of the team let you know that he already worked on the stories,  did start a UI and also went himself on holidays to France!
+
+>_"Hi backEndDevelloper,_
+
+>_Hope you had nice holidays.
+>I did create an UI and prepared resources calls for those 2 user stories.
+>You only should have to create the right endpoints for the application to call and everything should work fine!...
+>Unless I forgot something of course._
+
+>_A bientôt !"_
```
Comment (engagetech) [inline README.md]: needs to sign with robee's name

File: README.md (hunk 1 above)
Comment (alexmcbeth) [inline README.md]: As a user, i want to be able to enter my expenses and have them saved for later

File: README.md [hunk 3]
```diff
@@ -2,22 +2,91 @@ Goal
...
+> 1. _The date of the expense_
+> 2. _The value of the expense_
+> 3. _The reason of the expense_
+
+> _When I click "Save Expense", the expense is then saved in the database._
+> _The new expense can then be seen in the list of submitted expenses._
+
+By email, the front end developer of the team let you know that he already worked on the stories,  did start a UI and also went himself on holidays to France!
+
+>_"Hi backEndDevelloper,_
```
Comment (alexmcbeth) [inline README.md]: Developer, single l

File: README.md [hunk 4]
```diff
@@ -2,22 +2,91 @@ Goal
...
+
+> **As a user, I want to be able to save expenses in euros**
+
+> _As a user, in the UI, when i write an expense, I can add the chars_ `EUR` _after it (example : 12,00 EUR). 
+> When this happens, the application automatically converts the entered value into pounds and save the value as pounds in the database.
+The conversion needs to be accurate. It was decided that our application would call a public API to either realise the conversion or determine the right conversion rate, and then use it._
+
+**User story 4:**
+
+>**Technical : Calculate the VAT before saving the expenses to the database**
```
Comment (alexmcbeth) [inline README.md]: This needs work, we need to frame it in the style of a requirement. I want to see the VAT calculation update in real time as i enter my expenses? something like that

**Pull request merged**